docs/html/development/contributing.rst

```diff
@@ -110,8 +110,8 @@ public is concerned, typo fixes, white space modification, etc. To mark a PR
 as trivial a contributor simply needs to add a randomly named, empty file to
 the ``news/`` directory with the extension of ``.trivial.rst``. If you are on a
 POSIX like operating system, one can be added by running
-``touch news/$(uuidgen).trivial.rst``. On Windows, the same result can be
-achieved in Powershell using ``New-Item "news/$([guid]::NewGuid()).trivial.rst"``.
+``touch news/+$(uuidgen).trivial.rst``. On Windows, the same result can be
```

Comment:
@sbidoul FYI, fragments with the leading `+` are called “orphan” within Towncrier and so it's usually good to follow their conventions for this random stuff that might not need to link issue/PR numbers.

Reply:
TIL. Thanks!